### PR TITLE
Updating base.html

### LINKED ISSUE #57: Update navigation bar
Update the navigation bar
- register and login page will be shown before the user logged in.
- logout and profile page will be shown after the user logged in.

### PR DESCRIPTION
**Pull request by Leeadar**
- Updating navigation bar
- Adding register and login pages to the navbar
- After user logs in, the user sees profile and logout pages
- If the user try to reach http://127.0.0.1:8000/profile/ before logging in, the user redirect to login page. 
The URL will be http://127.0.0.1:8000/login/?next=/profile/

closes #57

### CODE REVIEW

File: pickATrip_django_apps/static/pickATrip_django_apps/main.css [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+.account-img {
+  height: 125px;
+  width: 125px;
+  margin-right: 20px;
+  margin-bottom: 16px;
+}
+
+.account-heading {
+  font-size: 2.5rem;
+}
```
Comment (Yarboa) [inline pickATrip_django_apps/static/pickATrip_django_apps/main.css:84]: Please insert new line
Avoid CSS, please continue with bootstrap
make the base.html visible 1st

File: pickATrip_django_apps/templates/pickATrip_django_apps/base.html [hunk 2]
```diff
@@ -1,62 +1,73 @@
...
-				{{ message }}
-			</div>
-		{% endfor %}
-	{% endif %}
-
-	
-	<div id ="content" class = "container">
-		{% block content %}{% endblock %}
-	</div>
+    <link rel="stylesheet" type="text/css" href="{% static 'pickATrip_django_apps/main.css' %}">
```
Comment (Yarboa) [inline pickATrip_django_apps/templates/pickATrip_django_apps/base.html:13]: Please avoid CSS

**Comment by Leeadar** [comment]
@Yarboa I deleted the CSS file and updated base.html

**Review by Yarboa — APPROVED** [review]
Looks good to me

**Review by guy9050 — APPROVED** [review]
lgtm

**Pull request merged**